api/v1beta1/v1beta2_condition_consts.go

```diff
@@ -0,0 +1,262 @@
+
+	// MachineSetRemediatingV1Beta2Condition surfaces details about ongoing remediation of the controlled machines, if any.
+	MachineSetRemediatingV1Beta2Condition = RemediatingV1beta2Condition
+
+	// MachineSetDeletingV1Beta2Condition surfaces details about ongoing deletion of the controlled machines.
+	MachineSetDeletingV1Beta2Condition = DeletingV1beta2Condition
+
+	// MachineSetPausedV1Beta2Condition is true if this MachineSet or the Cluster it belongs to are paused.
+	MachineSetPausedV1Beta2Condition = PausedV1beta2Condition
+)
+
+// Conditions that will be used for the MachineDeployment object in v1Beta2 API version.
+const (
+	// MachineDeploymentAvailableV1Beta2Condition is true if the MachineDeployment is not deleted, and it has minimum
+	// availability according to parameters specified in the deployment strategy, e.g. If using RollingUpgrade strategy,
+	// availableReplicas must be greater or equal than desired replicas - MaxUnavailable replicas.
+	MachineDeploymentAvailableV1Beta2Condition = AvailableV1beta2Condition
+
+	// MachineDeploymentMachinesReadyV1Beta2Condition surfaces detail of issues on the controlled machines, if any.
+	MachineDeploymentMachinesReadyV1Beta2Condition = MachinesReadyV1beta2Condition
+
+	// MachineDeploymentMachinesUpToDateV1Beta2Condition surfaces details of controlled machines not up to date, if any.
+	MachineDeploymentMachinesUpToDateV1Beta2Condition = MachinesUpToDateV1beta2Condition
+
+	// MachineDeploymentScalingUpV1Beta2Condition is true if available replicas < desired replicas.
+	MachineDeploymentScalingUpV1Beta2Condition = ScalingUpV1beta2Condition
+
+	// MachineDeploymentScalingDownV1Beta2Condition is true if replicas > desired replicas.
+	MachineDeploymentScalingDownV1Beta2Condition = ScalingDownV1beta2Condition
+
+	// MachineDeploymentRemediatingV1Beta2Condition details about ongoing remediation of the controlled machines, if any.
+	MachineDeploymentRemediatingV1Beta2Condition = RemediatingV1beta2Condition
+
+	// MachineDeploymentDeletingV1Beta2Condition surfaces details about ongoing deletion of the controlled machines.
+	MachineDeploymentDeletingV1Beta2Condition = DeletingV1beta2Condition
+
+	// MachineDeploymentPausedV1Beta2Condition is true if this MachineDeployment or the Cluster it belongs to are paused.
+	MachineDeploymentPausedV1Beta2Condition = PausedV1beta2Condition
+)
+
+// Conditions that will be used for the Cluster object in v1Beta2 API version.
+const (
+	// ClusterAvailableV1Beta2Condition is true if the Cluster's is not deleted, and RemoteConnectionProbe, InfrastructureReady,
+	// ControlPlaneAvailable, WorkersAvailable, TopologyReconciled (if present) conditions are true.
+	// If conditions are defined in spec.availabilityGates, those conditions must be true as well.
+	ClusterAvailableV1Beta2Condition = AvailableV1beta2Condition
+
+	// ClusterTopologyReconciledV1Beta2Condition True if the topology controller is working properly.
```

Comment:
```suggestion
	// ClusterTopologyReconciledV1Beta2Condition is true if the topology controller is working properly.
```
Might be worth the note that it's only set for classy clusters (the one you have on Cluster.status.conditions IIRC)

Reply:
Done